Add issues closed line graph and update hovertemplate

christinaexyou opened the pull request:
@cdolfi Resolves part of https://github.com/oss-aspen/8Knot/issues/468 and https://github.com/oss-aspen/8Knot/issues/470 by making the following changes:

* Adds line for issues closed
* Updates the hover template with Date, indicating the time window for which prolificacy is calculated

cdolfi reviewed (changes requested):
Lets add issue comments to this one too and rebase to main and will be good to go!

christinaexyou commented:
@cdolfi Changes made:
* Added issue comment line
* Updated legend labels to be more concise - changed `PR Review Comment` to `PR Comment` and `PR Request Review` to `PR Review`

cdolfi commented on 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:
```diff
@@ -18,8 +18,8 @@
 
-PAGE = "contributors"
-VIZ_ID = "contrib-prolificacy-over-time"
+PAGE = "contributors"  
```
remove spaces

cdolfi commented on 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:
```diff
@@ -60,67 +58,66 @@
-                                        ),
-                                    ],
+                                        [dcc.Slider(
+                                            id=f"threshold-{PAGE}-{VIZ_ID}",  
```
change back format

cdolfi commented on 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:
```diff
@@ -60,67 +58,66 @@
-                                    html_for=f"window-width-{PAGE}-{VIZ_ID}",
-                                    width="auto",
-                                ),
+                            [   dbc.Label(
```
same on format

cdolfi reviewed (changes requested):
Seems to be something weird happening format wise. Once that is fixed overall in the file I think it will be ready to merge (hard to see with current state, check out the "review" view if you want a good look)

cdolfi commented:
@christinaexyou let me know when this is ready for re-review. If there is not an obvious way of fixing file wide this @JamesKunstle might have some guidance

JamesKunstle commented:
@cdolfi @christinaexyou I'll pull this now and take a look to see if I have any ideas.

JamesKunstle commented:
Working on this, though I'm getting a DBReadFailure for some reason, so I may need to spend some time debugging tomorrow before I can actually review this PR.

JamesKunstle commented:
@christinaexyou 

## Review 

I pulled your most updated branch and tried the following orgs/repos, but neither 'Issues Closed' nor 'Issue Messages' lines appeared. 

'chaoss' org
'3scale' org
'kubernetes/kubernetes' repo

This is a screenshot of what I get for 'kubernetes/kubernetes':
<img width="749" alt="Screenshot 2023-10-04 at 1 58 47 PM" src="https://github.com/oss-aspen/8Knot/assets/52969093/66c32bb5-dc81-4dc5-a155-b2c155187372">

And this is my current branch's log:
<img width="668" alt="Screenshot 2023-10-04 at 2 01 02 PM" src="https://github.com/oss-aspen/8Knot/assets/52969093/57243fc4-804b-4be8-b7c6-ccfabd7a7f33">

Which I believe is up to date.


## One additional request

I'd love it if the Action Type was in the hover information. That'd make reading the graph much easier.

christinaexyou commented:
@cdolfi @JamesKunstle Changes made:
* Reverted formatting
* `Issue Closed` and `Issue Comment` line graphs are visible
* Shortened `PR Request ...` labels to `PR`
* Updated hover template to display action type

cdolfi reviewed (approved):
lgtm! Once james reviews we can merge

JamesKunstle commented:
@christinaexyou Yuh this looks great, gonna merge it.
